ckanext/data_qld/resource_freshness/helpers/helpers.py

```diff
@@ -36,8 +37,9 @@ def update_frequencies_from_config():
 
 def recalculate_next_update_due_date(flattened_data, update_frequency, errors, context):
     days = get_update_frequencies().get(update_frequency, 0)
-    # Recalculate the next_update_due always against todays date
-    due_date = dt.datetime.utcnow().date() + dt.timedelta(days=days)
+    # Recalculate the next_update_due always against today's date
+    today = dt.datetime.now(h.get_display_timezone())
```

Comment:
I just noticed there is a bug. It should be just getting the date without time. Will fix it now.
We are assuming users will be setting the `next_update_due` field date in `Australian/Brisbane` timezone, so when we auto recalculate this value we are making the same assumption to use `Australian/Brisbane`.
Because there is no time or timezone information stored there is nothing to convert and it will display as a date which is assumed `Australian/Brisbane`.